cli/src/config/templates.toml

```diff
@@ -91,33 +91,28 @@ if(root,
   )
 )
 '''
-builtin_log_compact = '''
+
+'builtin_log_compact_desc(full_description)' = '''
 if(root,
   format_root_commit(self),
   label(if(current_working_copy, "working_copy"),
     concat(
-      separate(" ",
-        format_short_change_id_with_hidden_and_divergent_info(self),
-        format_short_signature(author),
-        format_timestamp(committer.timestamp()),
-        bookmarks,
-        tags,
-        working_copies,
-        if(git_head, label("git_head", "git_head()")),
-        format_short_commit_id(commit_id),
-        if(conflict, label("conflict", "conflict")),
-      ) ++ "\n",
+      format_commit_header(self) ++ "\n",
       separate(" ",
         if(empty, label("empty", "(empty)")),
         if(description,
-          description.first_line(),
+          if(full_description, description, description.first_line()),
```

Comment:
nit: I wouldn't add `builtin_log_compact_desc()` function (which seems not so useful), and just copy-paste the second line.

Reply:
Is that what you meant?

Reply:
Yes, thanks.